appcues/src/main/java/com/appcues/ui/presentation/AppcuesViewModel.kt

```diff
@@ -21,15 +21,14 @@ import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
-import org.koin.core.component.KoinScopeComponent
-import org.koin.core.component.inject
-import org.koin.core.scope.Scope
 
 internal class AppcuesViewModel(
-    override val scope: Scope,
     private val renderContext: RenderContext,
+    private val coroutineScope: AppcuesCoroutineScope,
+    private val experienceRenderer: ExperienceRenderer,
+    private val actionProcessor: ActionProcessor,
     private val onDismiss: () -> Unit,
-) : ViewModel(), KoinScopeComponent {
```

Comment:
changed injection approach to constructor base to simplify test, decouple from the KOIN framework